### ISSUE TITLE
Tune hyperparameters (i.e. epochs, batch-size, learning rate, layers, neurons)

### ISSUE THREAD

**Issue by msn5265** [psu-nittanyaiadvance/VirusMitigation#30]
### Description of the Task:

Experiment with learning rate, batch size, and other hyperparameters

### Acceptance Criteria:

Optimal hyperparameter configuration is identified
